src/dfm/data/load_datasets.py

```diff
@@ -132,73 +157,162 @@ def load_danews(
-    danews = load_danews(columns_to_keep=columns_to_keep, path_to_danews=path_to_danews, repeats=1_000)
-    dagw_dfm = load_dagw_dfm(columns_to_keep=columns_to_keep, path_to_dagw=path_to_dagw, repeats=1_000)
-    hopetwitter = load_hopetwitter(columns_to_keep = columns_to_keep, path_to_hopetwitter=path_to_hopetwitter, repeats=1_000)
-    nat = load_nat(columns_to_keep=columns_to_keep, path_to_nat=path_to_nat, repeats=100)
-    train = interleave_datasets([danews["train"], dagw_dfm["train"], hopetwitter["train"], nat["train"]], probabilities=probabilities)
+    """
+    Loads Danish collosal corpus (DCC) version 1.
+
+    Args:
+        probabilities (Optional[List[float]], optional): Interleave probabilites of the
+            subdatasets. Defualts to {"danews": 0.06, "dagw_dfm": 0.06, "hopetwitter":
+            0.03, "nat": 0.85}.
+        n_training_repeats (Dict[str, int], optional): Number of times to repeat the
+            training dataset of each dataset.
+        path_to_hopetwitter (Union[str, Path], optional): path to Hopetwitter.
+            Defaults to /work/twitter_cleaned.
+        path_to_dagw (Union[str, Path], optional): path to DAGW_DFM.
+            Defaults to /work/dagw-clean.
+        path_to_danews (Union[str, Path], optional): path to DaNews.
+            Defaults to /work/hope-infomedia_cleaned.
+        path_to_nat (Union[str, Path], optional): path to NAT dataset.
+            Defaults to /work/netarkivet-cleaned.
+        columns_to_keep (List[str], optional): Columns to keep across the datasets.
+        kwargs: arguments to be passed forward to load_dataset
+
+    Returns:
+        IterableDatasetDict: A datasets IterableDatasetDict
+    """
+    versions_options = ["1.0.0"]
+    if version != "1.0.0":
+        raise ValueError(
+            "Version {version} is not available. Available version"
+            + f" include: {versions_options}"
+        )
+
+    danews = load_danews(
+        columns_to_keep=columns_to_keep,
+        path_to_danews=path_to_danews,
+        n_training_repeats=n_training_repeats["danews"],
+        **kwargs,
+    )
+    dagw_dfm = load_dagw_dfm(
+        columns_to_keep=columns_to_keep,
+        path_to_dagw=path_to_dagw,
+        n_training_repeats=n_training_repeats["dagw_dfm"],
+        **kwargs,
+    )
+    hopetwitter = load_hopetwitter(
+        columns_to_keep=columns_to_keep,
+        path_to_hopetwitter=path_to_hopetwitter,
+        n_training_repeats=n_training_repeats["hopetwitter"],
+        **kwargs,
+    )
+    nat = load_nat(
+        columns_to_keep=columns_to_keep,
+        path_to_nat=path_to_nat,
+        n_training_repeats=n_training_repeats["nat"],
+        **kwargs,
+    )
+    probabilities = [
+        probabilities[k] for k in ["danews", "dagw_dfm", "hopetwitter", "dfm"]
+    ]
+    train = interleave_datasets(
+        [danews["train"], dagw_dfm["train"], hopetwitter["train"], nat["train"]],
+        probabilities=probabilities,
+        **kwargs,
+    )
```

Comment:
Hmm, still order-dependent. E.g. if we change any item in the first list, we mismatch probabilities. I'd prefer to have e.g. a dict with `{"name": {"dataset": data, "probability": prob}}`, and then iterate over the items to unpack. 

Makes mismatching much less probable.